src/components/donation/Steps/Summary/Summary.tsx

```diff
@@ -37,7 +37,11 @@ export default function Summary({
       Amount={Amount}
       amount={amount}
       splitLiq={liquidSplitPct}
-      onBack={() => dispatch(setStep("tip"))}
+      onBack={() =>
+        recipient.hide_bg_tip
+          ? dispatch(setStep("splits"))
+          : dispatch(setStep("tip"))
+      }
```

Comment:
An alternative in this particular case would be to move this check inside the `setStep` action itself, but that would cause confusion when a dev intends to set the step to `"tip"`, but sees that the app doesn't allow him to do that. I opted for making it `setStep`'s responsibility to just set the step and not worry about which exact step to show.